Bottom: c16744a8a512154ceca49ec2745063d7ef7012b9
Top:    ee6ae72e89dcfc5bcb6c212d2f1ece15c56f9df8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-03 23:30:51 +0100

Refresh of file-format-extension-to-suppo

---

diff --git a/src/file.h b/src/file.h
index 64eba23..13e47b1 100644
--- a/src/file.h
+++ b/src/file.h
@@ -56,7 +56,7 @@ void sort_netlist (void);
  * guidance to the user as to what the minimum version of pcb required
  * is.
  */
-#define PCB_FILE_VERSION 20070407
+#define PCB_FILE_VERSION 20100603
 
 
 #ifndef HAS_ATEXIT
diff --git a/src/parse_y.y b/src/parse_y.y
index 15765f4..4677695 100644
--- a/src/parse_y.y
+++ b/src/parse_y.y
@@ -1100,11 +1100,11 @@ text_hi_format
 Polygon (SFlags) (
 @ @ @ @dots{} (X Y) @dots{}
 @ @ @ @dots{} [X Y] @dots{}
-Hole (
-@ @ @ @dots{} (X Y) @dots{}
-@ @ @ @dots{} [X Y] @dots{}
-)
-...
+@ @ @ Hole (
+@ @ @ @ @ @ @dots{} (X Y) @dots{}
+@ @ @ @ @ @ @dots{} [X Y] @dots{}
+@ @ @ )
+@ @ @ @dots{}
 )
 @end syntax
 
@@ -1113,6 +1113,8 @@ Hole (
 Symbolic or numeric flags.
 @item X Y
 Coordinates of each vertex.  You must list at least three coordinates.
+@item Hole (...)
+Defines a hole within the polygon's outer contour. There may be zero or more such sections.
 @end table
 
 %end-doc */
